Composer/packages/lib/indexers/src/dialogIndexer.ts

```diff
@@ -180,6 +180,7 @@ function parse(id: string, content: any) {
   const luFile = typeof content.recognizer === 'string' ? content.recognizer : '';
   const qnaFile = typeof content.recognizer === 'string' ? content.recognizer : '';
   const lgFile = typeof content.generator === 'string' ? content.generator : '';
+  const isFormDialog = has(content, '$Generator'); // mark as form generated dialog;
```

Comment:
We should leverage a more systematic to tell isFormDialog.

The correct way to tell isFormDialog is schema property is not null dialog.content?.schema !== undefined

Reply:
@boydc2014 updated to `const isFormDialog = has(content, 'schema');`